src/app/features/conditions/edit-conditions/conditions.component.html

```diff
@@ -109,6 +109,13 @@
               [config]="{ flags: QUEST_STATE, name: 'ConditionValue2' }"
               [modalClass]="'modal-lg'"
             ></keira-flags-selector-btn>
+			<keira-flags-selector-btn
+			  *ngIf="showReactionTo"
+              [control]="editorService.form.controls.ConditionValue2"
+              [config]="{ flags: RANKMASK, name: 'ConditionValue2' }"
+              [modalClass]="'modal-lg'"
+            ></keira-flags-selector-btn>
```

Comment:
code indentation is wrong

Reply:
Ah yes, I dont know why that got fked lol